webroot/js/directives/sentence-and-translations.dir.js

```diff
@@ -110,6 +120,26 @@
         vm.editSentence = editSentence;
         vm.favorite = favorite;
         vm.adopt = adopt;
+        vm.list = list;
+        vm.closeList = closeList;
+        vm.toggleList = toggleList;
+        vm.addToNewList = addToNewList;
+        vm.searchList = searchList;
+        vm.listType = 'of_user';
+        vm.show = show;
+        vm.hide = hide;
+
+        /////////////////////////////////////////////////////////////////////////
+
+        $scope.$on('newListCreated', function(event, data, sentenceId) {
+            var list = angular.copy(data);
+            $cookies.put('most_recent_list', list.id);
+            list.hasSentence = vm.sentence.id === parseInt(sentenceId);
+            list.is_mine = '1';
+            allLists.unshift(list);
+            resetLists();
+            moveRecentListToTop();
```

Comment:
I think there is no need to call moveRecentListToTop() here because resetLists() does that already.